learning/step_by_step/animations.rst

```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
 
-.. image:: img/animplayer.png
+.. image:: img/robisplash_anim_inspector.png
 
 When a node of this type is selected, the animation editor panel will
 appear:
 
-.. image:: img/animpanel.png
-
-So, it's time to create a new animation! Press the new animation button
-and name the animation "intro".
-
-.. image:: img/animnew.png
+.. image:: img/robisplash_anim_editor.png
 
-After the animation has been created, it's time to edit it, by
-pressing the "edit" button:
+The animation editor panel stays visible until manually hidden.
 
-.. image:: img/animedit.png
-
-Editing the animation
----------------------
+Creating the animation
+----------------------
 
-Now this is when the magic happens! Several things happen when the
-"edit" button is pressed, the first one is that the animation editor
-appears above the animation panel. (In Godot 2.x, this button has been
-removed, instead, click on the 'animation' toggle at the bottom right 
-for similar functionality.)
+It's time to create a new animation! Press the new animation button and name
+the animation "intro" when the dialog appears.
 
-.. image:: img/animeditor.png
+.. image:: img/robisplash_anim_new.png
 
-The second and most important, is that the property editor enters
+Now that we have an animation the property editor enters
 "animation editing" mode. In this mode, a key icon appears next to
 every property of the property editor. This means that, in Godot, *any
 property of any object* can be animated:
 
-.. image:: img/propertykeys.png
+.. image:: img/robisplash_anim_property_keys.png
 
-Making the logo appear
-----------------------
+Editing the animation
+---------------------
 
-The logo will appear from the top of the screen. After selecting
-the animation player, the editor panel will stay visible until
-manually hidden (or the animation node is erased). Taking advantage of
-this, select the "logo" node and go to the "pos" property, move it up
-to position: 114,-400.
+The logo will appear from the top of the screen.
 
-Once in this position, press the key button next to the property:
+With the animation editor panel open, select the "logo" node and set the "Rect / Position"
+property to: 118, -400.
 
-.. image:: img/keypress.png
+Once in this position, press the key button next to the property:
 
-As the track is new, a dialog will appear asking to create it. Confirm
-it!
+.. image:: img/robisplash_anim_logo_inspector_key.png
 
-.. image:: img/addtrack.png
+When the dialog appears, confirm that you are creating a new track.
 
 The keyframe will be added in the animation player editor:
 
-.. image:: img/keyadded.png
+.. image:: img/robisplash_anim_editor_keyframe.png
 
 Move the editor cursor to the end, by clicking here:
 
-.. image:: img/move_cursor.png
+.. image:: img/robisplash_anim_editor_track_cursor.png
 
-Change the logo position to 114,0 and add a keyframe again. With two
+Change the logo position to 118, 0 and add a keyframe again. With two
```

Comment:
In general, most docs highlight variables and values with code tags to make them more visible: ``(118, 0)``